components/MintNFT.vue

```diff
@@ -40,64 +40,75 @@ const route = useRoute()
 
 const step = ref(1)
 const error = ref('')
-const isLoading = ref(false)
+const isLoading = defineModel<boolean>('loading')
 
-const iscnIdInput = ref('')
 const iscnOwner = ref('')
 const iscnData = ref<any>(null)
 const classId = ref('')
 const liteMintNFTRef = ref<any>(null)
 
 const iscnId = computed(() => iscnData.value?.['@id'])
 
-const isFormValid = computed(() => {
-  return liteMintNFTRef.value?.isFormValid
-})
-
-const emit = defineEmits(['submit'])
 const showEditISCNModal = ref(false)
-const editISCNRef = ref<any>(null)
+const isFormValid = defineModel<boolean>('valid')
```

Comment:
Sorry for changing back-and-forth, but since `defineModel` allow parent to modify this variable, maybe we should just stick with `ref` + `watch` + `emit`

Reply:
[♻️ Use emit event instead of v-model](https://github.com/likecoin/nft-book-press/pull/344/commits/8cb618ad0a9826cf4898535f2c8e1184f314f7e8)